Homework6/Task1.py

```diff
@@ -0,0 +1,85 @@
+""" Create simulator from real life. This can be booking room in hotel,
+visit to casino, visit to bar. Create 3-4 objects, that can describe
+situation. Objects should contain attributes and methods to simulate
+some use cases. Completed program should print object states, it actions
+(methods) and objects interaction.
+    There is aggregation in created classes. Person can visit any bar
+spending his money, some objects can take debt, all classes communicate
+with each other and can count own money. """
+
+
+class Bar(object):
+    def __init__(self, name):
+        self.name = name
+        self.income = 0
+
+    def visited(self, visitor_money):
+        if visitor_money:
+            print('{}: Thank you!'.format(self.name))
+        else:
+            print('{}: Give us anything!'.format(self.name))
+        self.income += visitor_money
+
+    def total_income(self):
```

Comment:
our income isn't set to zero at the beginning at the next day and it seems this method shouldn't work as expected